filemage/README.md

```diff
@@ -0,0 +1,112 @@
+# Agent Check: filemage
+
+## Overview
+
+This check monitors [FileMage][1].
+
+## Setup
+
+### Building the package
+
+1. Install the [Datadog Developer Toolkit][3].
+
+2. Clone the `integrations-extras` repository:
+
+```shell
+git clone https://github.com/DataDog/integrations-extras.git
+```
+
+3. Update your `ddev` config with the repo path:
+
+```shell
+cd integrations-extras/
+ddev config set extras $(pwd)
+```
+
+4. Build the `datadog-filemage` package:
+
+```shell
+ddev -e release build filemage
+```
+
+### Installing the package
+
+Once you've built the wheel package, install it on a host:
+
+1. Ensure that the [Datadog Agent][2] is installed.
+
+2. Install the package:
+
+```shell
+chown dd-agent:dd-agent filemage/dist/datadog_filemage*.whl
+sudo -u dd-agent datadog-agent integration install -w filemage/dist/datadog_filemage*.whl
+```
+
+### Configuration
+
+1. Edit the `filemage.d/conf.yaml.example` file in the `conf.d/` folder at the root of your [Agent Configuration Directory][10] to start collecting your FileMage [metrics](#metrics).  
+   Once complete, save the modified file as `filemage.d/conf.yaml`.  
+   See the [sample filemage conf.yaml][5] for all available configuration options.
+
+2. [Restart the Agent][6].
+
+### Validation
+
+Run the [Agent's `status` subcommand][7] and look for `filemage` under the Running Checks section.
+
+```shell
+sudo -u dd-agent datadog-agent status
+```
```

Comment:
```suggestion

```

Reply:
Suggesting to remove this as it may be misleading for users with different commands -- users should reference the list linked in line 55.